filtered-resource-service/src/component-tests/java/uk/gov/gchq/palisade/component/filteredresource/web/AkkaWebsocketTest.java

```diff
@@ -50,15 +50,15 @@
 import uk.gov.gchq.palisade.service.SimpleConnectionDetail;
 import uk.gov.gchq.palisade.service.filteredresource.model.FilteredResourceRequest;
 import uk.gov.gchq.palisade.service.filteredresource.model.MessageType;
-import uk.gov.gchq.palisade.service.filteredresource.model.WebsocketMessage;
+import uk.gov.gchq.palisade.service.filteredresource.model.WebSocketMessage;
 import uk.gov.gchq.palisade.service.filteredresource.repository.TokenOffsetController;
 import uk.gov.gchq.palisade.service.filteredresource.repository.TokenOffsetController.TokenOffsetCommand;
 import uk.gov.gchq.palisade.service.filteredresource.repository.TokenOffsetPersistenceLayer;
-import uk.gov.gchq.palisade.service.filteredresource.service.WebsocketEventService;
+import uk.gov.gchq.palisade.service.filteredresource.service.WebSocketEventService;
 import uk.gov.gchq.palisade.service.filteredresource.stream.config.AkkaRunnableGraph.AuditServiceSinkFactory;
 import uk.gov.gchq.palisade.service.filteredresource.stream.config.AkkaRunnableGraph.FilteredResourceSourceFactory;
 import uk.gov.gchq.palisade.service.filteredresource.web.AkkaHttpServer;
-import uk.gov.gchq.palisade.service.filteredresource.web.router.WebsocketRouter;
+import uk.gov.gchq.palisade.service.filteredresource.web.router.WebSocketRouter;
```

Comment:
Could not get this to link to line 81, the class name so I put it here.  Suggest you use the camel case with the test classes as well.

Reply:
Updated Websocket within this file where appropriate to WebSocket